packages/create-cloudflare/src/frameworks/next/index.ts

```diff
@@ -140,11 +141,16 @@ const config: FrameworkConfig = {
 	generate,
 	configure,
 	displayName: "Next",
-	packageScripts: {
-		"pages:build": `${dlx} @cloudflare/next-on-pages@1`,
-		"pages:deploy": `${npm} run pages:build && wrangler pages deploy .vercel/output/static`,
-		"pages:watch": `${npx} @cloudflare/next-on-pages@1 --watch`,
-		"pages:dev": `${npx} wrangler pages dev .vercel/output/static --compatibility-flag=nodejs_compat`,
+	getPackageScripts: async () => {
+		const nextOnPagesScope = ["npm", "bun"].includes(npm) ? "@cloudflare/" : "";
+		const nextOnPagesCommand = `${nextOnPagesScope}next-on-pages`;
+		const pmCommand = npm === "npm" ? "npm run" : npm;
```

Comment:
How about we add a `run` to `detectPackageManager` so we can reat it like `npx` and `dlx`?

Reply:
no sorry, the logic you reviewed was wrong, that's my bad sorry for the confusion 😓